### PR TITLE
[README]: Improve Chinese translation

### PR DESCRIPTION
**Pull request by diskdance**
The reason of renaming is that Chinese language has two varieties: Simplified Chinese (zh-CN) and Traditional Chinese (zh-TW). Using `zh` is too ambiguous because usually people from Mainland China would like to use the former while people from Taiwan tend to use the latter. The Chinese translation of readme is actually in Simplified Chinese, so I think `zh-CN` suits better here.

### CODE REVIEW

**Comment by sandersn** [comment]
Are there any other readers/writers of simplified Chinese who can review the edits?

**Comment by Josh-Cena** [comment]
Quick heads-up: `zh-CN` is not recommended, because it's not listed in the [IANA language subtag registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry). You should use `zh-Hans` instead, and `zh-Hans-CN` if you want to be exact.

File: README.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,607 @@
+# Definitely Typed
+
+> 提供*高质量* TypeScript 类型定义的仓库。
+
+*您也可以阅读此自述文件的[英语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md)、[西班牙语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.es.md)、[韩语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ko.md)、[俄罗斯语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ru.md)、[葡萄牙语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.pt.md)、[意大利语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.it.md)或[日语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ja.md)版本！*
```
Comment (Josh-Cena) [inline README.zh-CN.md:5]:
```suggestion
*你也可以阅读此 README 文件的[英语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md)、[西班牙语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.es.md)、[韩语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ko.md)、[俄罗斯语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ru.md)、[葡萄牙语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.pt.md)、[意大利语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.it.md)或[日语](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.ja.md)版本！*
```

([Microsoft translation guidelines](https://learn.microsoft.com/en-us/globalization/localization/ministyleguides/mini-style-guide-sim-chinese) suggest using 「你」 over 「您」 where possible)

**Review by Josh-Cena — COMMENTED** [review]
Frankly I find the previous zh translation quite readable and preferable in terms of tone and register. It should be a good starting point for any further revisions. To help with review, I suggest you rebase the PR and split it into two commits: the first one where you rename it to `zh-Hans.md` with no edits, and the second one where you do all the edits, so reviewers can compare the two versions side-by-side. It's quite hard for me (or for others) to figure out what's going on.

File: README.zh-CN.md (hunk 1 above)
Comment (diskdance) [reply README.zh-CN.md]: Thanks for the suggestion. I have changed it to "你".

**Comment by diskdance** [comment]
We need to update the text of the links to zh-Hans doc in the other docs with help from people using the corresponding languages.

**Review by sandersn — CHANGES_REQUESTED** [review]
The link update missed the .es.md and .pt.md files.
(I noticed because those are the easiest versions for me to read besides English.)

**Comment by diskdance** [comment]
> The link update missed the .es.md and .pt.md files. (I noticed because those are the easiest versions for me to read besides English.)

It looks like .es.md and .pt.md do not have link to other README files, and I cannot read in those languages either, so I skipped them intentionally.

**Comment by diskdance** [comment]
Oops, there is already https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63899. How should we handle it in this circumstance?

**Comment by sandersn** [comment]
The structure part looks good to me now, but still needs a sign off from a Chinese speaker.

**Comment by lakejason0** [comment]
I am a Chinese speaker, and I've been taking part in many localization projects (as in my profile, Minecraft and its related projects, paimon.moe, also Vanilla OS). This PR is valid, as there *is* quite a big difference between Simplified Chinese and Traditional Chinese due to regional word differences, and if we don't point it out, it make cause confusion and discouragement of reading the docs. Doing this before a Traditional Chinese version is made is also encouraging native Traditional Chinese users to actually contribute a native Traditional Chinese version IMO.

**Review by septs — APPROVED** [review]

**Comment by septs** [comment]
Why rename `README.zh.md` to `README.zh-Hans.md`?

Planning to add zh-Hant edition?

**Comment by septs** [comment]
suggest, revert the rename filename operation; split rename filename operation to new pull request.

github diff processor can't show improvement part diffs

**Comment by diskdance** [comment]
> suggest, revert the rename filename operation; split rename filename operation to new pull request.
> 
> github diff processor can't show improvement part diffs

It is visible at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64116/commits/3c21f494cbf63d3ca8071b5d6dc934d13e0a85ca.

**Review by septs — APPROVED** [review]

**Review by sandersn — APPROVED** [review]

**Pull request merged**